server/src/model.ts

```diff
@@ -3147,6 +3340,27 @@ export async function getPublicFolderContent({
       ownerId: true,
       name: true,
       imagePath: true,
+      createdAt: true,
```

Comment:
Delete `createdAt` and `lastEdited`, as they aren't in the type `ContentStructure`. Also on line 3396.